File: prime/prime.php
<? 
function main() {
	$res = array(3);
	$last = 3;
	while(true) {
		$last += 2;
		$prime = true;
		for($z=0;$z<sizeof($res);++$z) {
			$v = $res[$z];
			if($v*$v > $last) break;
			if($last%$v == 0) {
				$prime = false;
				break;
			}
		}
		if($prime) {
			$res[] = $last;
			if(sizeof($res)%100000 == 0) echo $last."\n";
			if($last > 9999999) break;
		}
	}
}
main();

File: prime/prime.py
res = [3]
last = 3
while(True):
	last += 2
	prime = True
	for v in res:
		if v*v > last: break
		if last%v == 0:
			prime = False
			break
	if prime:
		res.append(last)
		if len(res)%100000 == 0: print(last)
		if last > 9999999: break
